### DISCUSSION TITLE
Hide items from stars page after adding to list.

### DISCUSSION THREAD

**Discussion by rakibdev** [community/community#77777] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Treat lists as folders and stars as normal bookmarks. So if I add items to list. It should move the item  and not keep duplicate entry in stars page.

That way user can also tell how many stars are uncategorized/unlisted.

**Comment by Aman3899** [comment]
Okay, so you're saying when you add a starred item to a list, you want it to disappear from your main "Starred" page? Like moving a file into a folder, so it's no longer just sitting on your desktop.

That makes sense! It helps you keep things organized and easily see what's still "uncategorized" or needs to be sorted into a list. Basically, the "Starred" page would become a visual reminder of what still needs your attention. Good idea!
